egs2/TEMPLATE/svs1/svs.sh

```diff
@@ -0,0 +1,1174 @@
+#!/usr/bin/env bash
+
+# Credit to espnet asr.sh
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+min() {
+  local a b
+  a=$1
+  for b in "$@"; do
+      if [ "${b}" -le "${a}" ]; then
+          a="${b}"
+      fi
+  done
+  echo "${a}"
+}
+SECONDS=0
+
+# General configuration
+stage=1              # Processes starts from the specified stage.
+stop_stage=10000     # Processes is stopped at the specified stage.
+skip_data_prep=false # Skip data preparation stages.
+skip_train=false     # Skip training stages.
+skip_eval=false      # Skip decoding and evaluation stages.
+skip_upload=true     # Skip packing and uploading stages.
+ngpu=1               # The number of gpus ("0" uses cpu, otherwise use gpu).
+gpu_id=0             # GPU_id, only works when ngpu=1
+num_nodes=1          # The number of nodes.
+nj=32                # The number of parallel jobs.
+inference_nj=32      # The number of parallel jobs in decoding.
+gpu_inference=false  # Whether to perform gpu decoding.
+dumpdir=dump         # Directory to dump features.
+expdir=exp           # Directory to save experiments.
+python=python3       # Specify python to execute espnet2 commands.
+
+# Data preparation related
+local_data_opts="" # Options to be passed to local/data.sh.
+
+# Feature extraction related
+feats_type=raw       # Feature type (fbank or stft or raw).
+audio_format=wav    # Audio format: wav, flac, wav.ark, flac.ark  (only in feats_type=raw).
+min_wav_duration=0.1 # Minimum duration in second.
+max_wav_duration=20  # Maximum duration in second.
+use_sid=false        # Whether to use speaker id as the inputs (Need utt2spk in data directory).
+use_lid=false        # Whether to use language id as the inputs (Need utt2lang in data directory).
+use_xvector=false    # Whether to use x-vector (Require Kaldi).
```

Comment:
There is no need to require Kaldi now, you can refer to TTS template for a fix here

Reply:
TTS template keeps these opts. I haven't remove them in this commit.

Reply:
I will update the TTS template shortly. 
```suggestion
use_xvector=false    # Whether to use x-vector 
```